### DISCUSSION TITLE
Virus?

### DISCUSSION THREAD

**Discussion by Donermo** [Official-Husko/NN-Downloader#25] [Q&A] [answered]
On an online virus scanning site it says that this is a virus and grabs information and infects the system, i hope this isn't the case and that you can assure me that. Thanks

**Comment by Official-Husko** [comment, marked as answer]
its a false positive from the fact that i am packaging the program using UPX to make it smaller and more compact, because it uses a[ bat file auto updater system](https://github.com/Official-Husko/NN-Downloader/blob/master/modules/auto_update.py) and due to the sites it contacts which are for example the [proxy providers](https://github.com/Official-Husko/NN-Downloader/blob/master/modules/proxyScraper.py) if you use that. Feel free to browse through the source code if you are unsure or to build it yourself. In the future i plan to build it using github actions to keep the entire build process as transparent as possible. I'm looking into a new auto updater method which should hopefully avoid these false positives in the future.

Alternatively you can get yourself python from their [official website](https://www.python.org/downloads/release/python-3119/) and then download the source code and run it like that out of the box which even enables you to edit the code directly if you so please.


If you have any further questions or concerns feel free to ask away.

**Reply by Donermo** [reply]
Thanks
